graylog2-server/src/main/java/org/graylog2/indexer/IndexMapping.java

```diff
@@ -37,8 +37,7 @@
         return ImmutableMap.of(
                 "properties", partFieldProperties(analyzer),
                 "dynamic_templates", partDefaultAllInDynamicTemplate(),
-                // Compress source field
-                "_source", enabledAndCompressed(),
+                "_source", enabled(),
```

Comment:
This field is compressed by default now: https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking_20_mapping_changes.html#_compress_and_compress_threshold